ipaserver/install/server/upgrade.py

```diff
@@ -79,11 +79,12 @@ def uninstall_ipa_memcached():
     We can't use the full service uninstaller because that will attempt
     to stop and disable the service which by now doesn't exist. We just
     want to clean up sysrestore.state to remove all references to
-    ipa_kpasswd.
+    ipa_memcached.
     """
     ipa_memcached = service.SimpleServiceInstance('ipa_memcached')
 
     enabled = not ipa_memcached.restore_state("enabled")
+    ipa_memcached.restore_state("running")
 
     if enabled is not None and not enabled:
```

Comment:
Totally OT: `enabled` will never be None so the first part of the condition is redundant.